src/en/charms-bundles.md

```diff
@@ -243,36 +243,28 @@ applications:
             cert: include-base64://my-cert.crt
 ```
 
-Furthermore, the `--bundle-config` option can be used when you want to use a
-standard bundle but keep model-specific configuration in a separate file.  Any
-values specified for an application in the file override those values defined
-in the bundle, with the exception of the map type values, where the maps are
-merged with preference given to the bundle-config file. The file can make use
-of the `include-file://` and `include-base64://` directives mentioned above for
-local bundles (for options and annotation values). A file location can be
-expressed with an absolute or relative (to the bundle-config file) path. For
-example:
+## Overlay bundles
```

Comment:
Hmm, okay. I think at some point this needs a proper worked example - e.g. a bundle, an example overlay, and what you would end up with. Maybe you could add that as a TODO

Reply:
Done.